src/components/views/voip/CallView.tsx

```diff
@@ -429,7 +429,7 @@ export default class CallView extends React.Component<IProps, IState> {
                         transferee: transfereeName,
                     },
                     {
-                        a: sub => <AccessibleButton kind="link" onClick={this.onTransferClick}>
+                        a: sub => <AccessibleButton kind="link_inline" onClick={this.onTransferClick}>
```

Comment:
See above: `"Consulting with %(transferTarget)s. <a>Transfer to %(transferee)s</a>",`